### PR TITLE
Add streaming notes to update example file

### PR DESCRIPTION
**Pull request by vmatyusGitHub**
## PR summary

Fixes: <! -- link to issue -->

**Note: An existing issue is [required](https://github.com/IBM/cloudant-python-sdk/blob/master/CONTRIBUTING.md#PRs) before opening a PR.**

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [ ] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [x] Documentation content changes
- [ ] Other (please describe)

## What is the new behavior?

Add stream handling examples in note comments to update document example.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by mojito317 — APPROVED** [review]
LGTM! +1

File: test/examples/src/update_doc.py [hunk 1]
```diff
@@ -34,6 +40,12 @@
         document=document
     ).get_result()
 
+    # Note: for byte request (Input Stream) use:
+    # update_document_response = client.post_document(
+    #     db=example_db_name,
+    #     document=inputStream
```
[the remark this answers is not in the document]
Comment (vmatyusGitHub) [reply test/examples/src/update_doc.py:46]: There is already an example at the return value of `get_document_as_stream`, in this current example this is `documentAsStream`. WDYT about replacing the `inputStream` with `documentAsStream` variable?

[the remark this answers is not in the document]
Comment (emlaver) [reply test/examples/src/update_doc.py:46]: That may add some confusion. `documentAsStream` contains the current document in the database.  After this call the `document` non-streaming example is modified (add `address`, remove `joined`) and `documentAsStream` stays the same.
I think we can either a) keep this as-is with `inputStream` and hope the user can figure out how to create this variable or b) show how to: i) transform `documentAsStream` to a `dict` object ii) modify the doc just like `document` ii) convert the modified document back to `BinaryIO` object.

[the remark this answers is not in the document]
Comment (emlaver) [reply test/examples/src/update_doc.py:46]: Ok 👍 sounds good

File: test/examples/src/update_doc.py [hunk 2]
```diff
@@ -21,6 +21,12 @@
         doc_id=example_doc_id
     ).get_result()
 
+    # Note: for byte response (Output Stream) use:
```
[the remark this answers is not in the document]
Comment (vmatyusGitHub) [reply test/examples/src/update_doc.py]: Changed in 5962967

File: test/examples/src/update_doc.py [hunk 3]
```diff
@@ -34,6 +40,12 @@
         document=document
     ).get_result()
 
+    # Note: for byte request (Input Stream) use:
```
[the remark this answers is not in the document]
Comment (vmatyusGitHub) [reply test/examples/src/update_doc.py]: Changed in 5962967

File: test/examples/src/update_doc.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vmatyusGitHub) [reply test/examples/src/update_doc.py]: Considering the term change, I changed this too: 5962967

**Comment by emlaver** [comment]
@vmatyusGitHub can we update the `byte stream` comments in the other SDKs?

**Comment by vmatyusGitHub** [comment]
Updated also other language examples and unified the variable names to `documentAsByteStream`:

node: 
* https://github.com/IBM/cloudant-node-sdk/pull/267/commits/b81b399dcd419e3f9c3a18b7153a936a9caca92c
* https://github.com/IBM/cloudant-node-sdk/pull/267/commits/4e3978298dd0c92ec4db77d09a611e54b56786c6

go: 
* https://github.com/IBM/cloudant-go-sdk/pull/140/commits/23a277d9c76f962fa59d1111bdb5be4abfb5bacf

java: 
* https://github.com/IBM/cloudant-java-sdk/pull/117/commits/3fff1fe1f9aa89e902cbd6d71c8051d805030c12

**Comment by vmatyusGitHub** [comment]
Thank you for reviewing! 🙂

**Pull request merged**